apps/src/maker/ui/SetupChecklist.jsx

```diff
@@ -361,12 +357,9 @@ class SetupChecklist extends Component {
   installFirmware() {
     let firmataMarkdown;
     if (this.state.boardTypeDetected === BOARD_TYPE.MICROBIT) {
-      firmataMarkdown = applabI18n.makerSetupInstallFirmataMB({
-        firmataURL: MICROBIT_FIRMATA_URL,
```

Comment:
This URL param is no longer needed since we implemented the install micro:bit Firmata via clicking of button.
https://github.com/code-dot-org/code-dot-org/blob/69488763ae61cf4e7cb8d5196f3d618bf8e6898e/apps/i18n/applab/en_us.json#L490